### PR TITLE
[keras/preprocessing] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/preprocessing/text.py [hunk 1]
```diff
@@ -154,11 +154,11 @@ def hashing_trick(
     Args:
         text: Input text (string).
         n: Dimension of the hashing space.
-        hash_function: defaults to python `hash` function, can be 'md5' or
+        hash_function: when None uses a python `hash` function, can be 'md5' or
```
Comment (fchollet) [inline keras/preprocessing/text.py:157]: Please add a comma after `None` and use backticks. Also a period before "can"

Comment (haifeng-jin) [inline keras/preprocessing/text.py:157]: when -> When

**Review by haifeng-jin — CHANGES_REQUESTED** [review]

File: keras/preprocessing/text.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/preprocessing/text.py:157]: Done

Comment (SamuelMarks) [reply keras/preprocessing/text.py:157]: Done

File: keras/preprocessing/image.py [hunk 2]
```diff
@@ -1586,8 +1586,8 @@ def flow_from_directory(
               in the generator. See [this script](
               https://gist.github.com/fchollet/0830affa1f7f19fd47b06d4cf89ed44d)
               for more details.
-            target_size: Tuple of integers `(height, width)`, defaults to `(256,
-              256)`. The dimensions to which all images found will be resized.
+            target_size: Tuple of integers `(height, width)`. The dimensions to
+             which all images found will be resized. Defaults to `(256,256)`.
```
Comment (haifeng-jin) [inline keras/preprocessing/image.py:1590]: The indentation of this line doesn't seem right. You can temporarily use 2 spaces.

**Review by haifeng-jin — CHANGES_REQUESTED** [review]

File: keras/preprocessing/image.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/preprocessing/image.py]: Ok @haifeng-jin I updated everything in that `flow_from_directory` docstring to be 2-spaced and have `Defaults to` correctly set

**Review by haifeng-jin — APPROVED** [review]

**Pull request merged**